### PR TITLE
EC-63 repack Health SDK into Merchant SDK

### PR DESCRIPTION
**Pull request by motokotomax**

### CODE REVIEW

**Review by zladzeyka — APPROVED** [review]

**Review by zladzeyka — CHANGES_REQUESTED** [review]
@motokotomax could you please look at [Sonar](https://sonarcloud.io/project/issues?id=gini_gini-mobile-ios&pullRequest=543&resolved=false&sinceLeakPeriod=true) cloud issues?

File: MerchantSDK/GiniMerchantSDK/Documentation/source/Installation.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+```swift
+dependencies: [
+    .package(url: "https://github.com/gini/merchant-sdk-ios.git", .exact("1.0.0"))
+]
+```
+
+In case that you want to use [the certificate pinning](https://www.ssl.com/blogs/what-is-certificate-pinning/#:~:text=Certificate%20pinning%20is%20a%20security,(Transport%20Layer%20Security)%20protocols.) in the library, add `GiniHealthAPILibraryPinning`:
+```swift
+dependencies: [
+    .package(url: "https://github.com/gini/merchant-sdk-pinning-ios.git", .exact("4.1.0"))
```
[the remark this answers is not in the document]
Comment (motokotomax) [reply MerchantSDK/GiniMerchantSDK/Documentation/source/Installation.md]: fixed

File: MerchantSDK/GiniMerchantSDK/Package-release.swift [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
+        
+        .target(
+            name: "GiniMerchantSDK",
+            dependencies: ["GiniHealthAPILibrary"]),
+        .testTarget(
+            name: "GiniMerchantSDKTests",
+            dependencies: ["GiniHealthSDK"]),
```
[the remark this answers is not in the document]
Comment (motokotomax) [reply MerchantSDK/GiniMerchantSDK/Package-release.swift]: fixed

File: MerchantSDK/GiniMerchantSDK/Package-release.swift [hunk 3]
```diff
@@ -0,0 +1,32 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "GiniMerchantSDK",
+    defaultLocalization: "en",
+    platforms: [.iOS(.v12)],
```
[the remark this answers is not in the document]
Comment (motokotomax) [reply MerchantSDK/GiniMerchantSDK/Package-release.swift]: fixed

File: HealthSDK/GiniHealthSDK/GiniHealth_Logo.png
[the remark this answers is not in the document]
Comment (motokotomax) [reply HealthSDK/GiniHealthSDK/GiniHealth_Logo.png:1]: thanks, yes you right - recover icon

**Review by zladzeyka — APPROVED** [review]

**Review by zladzeyka — APPROVED** [review]
👍🏻

**Pull request merged**